hammr/utils/publish_builders.py

```diff
@@ -126,45 +126,49 @@ def publish_susecloud(builder):
     return pimage
 
 
-def publish_openstack(builder):
+def publish_openstack(builder, keystone_version):
```

Comment:
if we follow the idea, this becomes:
```suggestion
def publish_openstack(builder, cred_account):
```